app/admin/dashboard.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register_page 'Dashboard' do
+  menu priority: 1, label: proc { I18n.t('active_admin.dashboard') }
+
+  content title: proc { I18n.t('active_admin.dashboard') } do
+    div class: 'blank_slate_container', id: 'dashboard_default_message' do
+      span class: 'blank_slate' do
+        span I18n.t('active_admin.dashboard_welcome.welcome')
+        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+      end
+    end
+
```

Comment:
lets don't  leave comment. May be here display list of users with command (block/unblock) ?

Reply:
This comment was generated by active_admin. The ability to block users is not yet added